docs/src/main/asciidoc/cache.adoc

```diff
@@ -598,11 +598,4 @@ the exception bubble out of the called method and catch it at the caller side:
 When the call to the remote throws an exception, the cache does not store the result,
 so that a subsequent call to `callRemote()` with the same parameter value will not be
 answered out of the cache.
-It will instead result in another
-attempt to call the remote.
-
-The previous code example has the side-effect that the cache logs the thrown exception
-with a long stack trace to the console. This is done to inform developers that something
-exceptional has happened, but if you have a setup like above, you are already catching
-the exception and know what you are doing.
```

Comment:
This was true at some point in the past but is no longer true today. It can be verified by checking the server log from `io.quarkus.cache.test.runtime.ThrowExecutionExceptionCauseTest` which throws exceptions from cached methods.